addons/l10n_fr_account/i18n/fr.po

```diff
@@ -20,909 +20,1066 @@ msgstr ""
+"3310-CA3G"
+msgstr ""
+"AB - Total à payer acquitté par la société tête de groupe sur la déclaration"
+" récapitulative 3310-CA3G"
 
-#. module: l10n_fr
-#: model:ir.model,name:l10n_fr.model_account_chart_template
+#. module: l10n_fr_account
+#: model:ir.model,name:l10n_fr_account.model_account_chart_template
 msgid "Account Chart Template"
 msgstr "Modèle de plan comptable"
 
-#. module: l10n_fr
-#: model:account.report.line,name:l10n_fr.tax_report_decompte_tva
+#. module: l10n_fr_account
+#: model:account.report.line,name:l10n_fr_account.tax_report_decompte_tva
 msgid "B. Settlement of VAT to be paid"
 msgstr "B. Décompte de la TVA à payer"
 
-#. module: l10n_fr
-#: model:account.report.line,name:l10n_fr.tax_report_B1
+#. module: l10n_fr_account
+#: model:account.report.line,name:l10n_fr_account.tax_report_B1
 msgid "B1 - Releases for consumption of petroleum products"
 msgstr "B1 - Mises à la consommation de produits pétroliers"
 
-#. module: l10n_fr
-#: model:account.report.line,name:l10n_fr.tax_report_B2
+#. module: l10n_fr_account
+#: model:account.report.line,name:l10n_fr_account.tax_report_B2
 msgid "B2 - Intra-Community acquisitions"
 msgstr "B2 - Acquisitions intracommunautaires"
 
-#. module: l10n_fr
-#: model:account.report.line,name:l10n_fr.tax_report_B3
-msgid "B3 - Taxable supplies of electricity, natural gas, heat or cooling in France"
-msgstr "B3 - Livraisons d'électricité, de gaz naturel, de chaleur ou de froid imposables en France"
+#. module: l10n_fr_account
+#: model:account.report.line,name:l10n_fr_account.tax_report_B3
+msgid ""
+"B3 - Taxable supplies of electricity, natural gas, heat or cooling in France"
+msgstr ""
+"B3 - Livraisons d'électricité, de gaz naturel, de chaleur ou de froid "
+"imposables en France"
 
-#. module: l10n_fr
-#: model:account.report.line,name:l10n_fr.tax_report_B4
-msgid "B4 - Purchases of goods or services from a taxable person not established in France"
-msgstr "B4 - Achats de bien ou de prestations de services réalisés auprès d'un assujetti non établi en France"
+#. module: l10n_fr_account
+#: model:account.report.line,name:l10n_fr_account.tax_report_B4
+msgid ""
+"B4 - Purchases of goods or services from a taxable person not established in"
+" France"
+msgstr ""
+"B4 - Achats de bien ou de prestations de services réalisés auprès d'un "
+"assujetti non établi en France"
 
-#. module: l10n_fr
-#: model:account.report.line,name:l10n_fr.tax_report_B5
+#. module: l10n_fr_account
+#: model:account.report.line,name:l10n_fr_account.tax_report_B5
 msgid "B5 - Regularisations"
 msgstr "B5 - Régularisations"
 
-#. module: l10n_fr
-#: model:account.report.column,name:l10n_fr.tax_report_balance
+#. module: l10n_fr_account
+#: model:account.report.column,name:l10n_fr_account.tax_report_balance
 msgid "Balance"
-msgstr "Balance"
+msgstr ""
+
+#. module: l10n_fr_account
+#: model_terms:ir.ui.view,arch_db:l10n_fr_account.fec_export_wizard_view
+msgid "Cancel"
+msgstr ""
```

Comment:
Can you please translate all the missing fields ?

They are a few in the file.

Reply:
@rugo-odoo The translation problems seems to have been fixed in this PR https://github.com/odoo/odoo/pull/172084
Maybe this PR (and it's corresponding task) should be canceled?